java/code/webapp/WEB-INF/pages/systems/sdc/overview.jsp

```diff
@@ -146,6 +146,7 @@
                 </c:when>
                 <c:otherwise>
                   <c:out value="${system.runningKernel}" />
+                  <a href="/rhn/manager/contentmanagement/filters?openFilterId=-1&openTemplate=LivePatchingSystem&systemId=${system.id}&systemName=${system.name}&kernelName=${system.runningKernel}">(<bean:message key="sdc.details.overview.create_clm_filter"/>)</a>
```

Comment:
Does this link need to be shown conditionally or is it fine to always show this?

Reply:
Hmm, good question. Not every user has live patching products, but the filter template itself is available unconditionally, so I'd say let's keep it like this unless we hear any feedback against it.